src/lib/modules/console/index.ts

```diff
@@ -72,14 +72,12 @@ class Console {
     });
   }
 
-  private processEmbarkCmd(cmd: string) {
+  private processEmbarkCmd(cmd: string, helpDescriptions: any[]) {
```

Comment:
Why don't you create an interface instead of using any.
in generate any === bad